### ISSUE TITLE
Button: primary color style does not meet contrast ratio

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#1584]
![Screen Shot 2021-01-29 at 9 21 18 AM](https://user-images.githubusercontent.com/226356/106300228-6f87bf00-6213-11eb-89a3-0dea91cd4980.png)

**Comment by sweeting** [comment]
Closed by #1593
